app/models/relationship.rb

```diff
@@ -0,0 +1,4 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: User.name
```

Comment:
Sao ở đây lại cần có class_name

Reply:
Dạ ở là một User có thể có nhiều người khác theo dõi và ngược lại ( many to many). Thì nó tạo ra một ạo một model trung gian lưu trữ quan hệ. class_name là cho phép bạn sử dụng một lớp khác.